errorhandling.js

```diff
@@ -5,6 +5,8 @@ exports.notAPath = (req, res) => {
 exports.customErrors = (err, req, res, next) => {
     if (err.status === 404) {
         res.status(404).send({msg: 'not found'})
+    } else if (err.status === 400) {
```

Comment:
You could tidy this up a bit by checking if the err.status prop exist then doing `res.status(err.status).send(err.msg)`

This way you can reject with whatever status and msg you want and it'll work for everything :)